sdk/rust/quickstart.mdx

```diff
@@ -11,41 +11,49 @@ In this Rust quickstart we will learn how to:
 - Execute a query using SQL
 
 <Steps>
-  <Step title="Install libsql-client">
+  <Step title="Install libsql crate">
 
-First begin by installing the `libsql-client` [crate](https://crates.io/crates/libsql-client):
+First begin by installing the `libsql` [crate](https://crates.io/crates/libsql):
 
 ```bash
-cargo add libsql-client
+cargo add libsql
 ```
 
   </Step>
 
   <Step title="Set environment variables">
-    Next export your `LIBSQL_CLIENT_URL` and `LIBSQL_CLIENT_TOKEN`:
+    Next export your `TURSO_DATABASE_URL` and `TURSO_AUTH_TOKEN`:
```

Comment:
I like that we encourage this. Perhaps an improvement we can make with other quickstarts.

Reply:
Consistency in the docs and references is good.